Assets/MixedRealityToolkit/Providers/UnityInput/MouseController.cs

```diff
@@ -99,16 +81,6 @@ public void Update()
                     }
                 }
 
-                if (Interactions[i].InputType == DeviceInputType.Scroll)
```

Comment:
>removed mouse wheel scrolling that modified mouse cursor position (not needed as spatial cursor is snapping to interactable objects)

Why was this completely removed?

Why not just update the input action that was driving it? Now the scroll cannot be used by anyone looking to configure it.

Reply:
pointing to my comment to luis above who was asking a similar question. The code how it used to be would just modify the x/y position values which was wrong and didn't work. I couldn't figure out what the intent behind this was. So even if you would've modified the controller mapping for this it would have in best case moved the mouse sideways and not perform any actual scroll action. There's no real use case for scroll in the mouse pointer how it's implemented now as we're snapping to all the interactable objects - without this feature it would make sense to move the cursor in depth with the scroll wheel.  Which input event would you recommend to send for the wheel that makes sense?